worker/worker.js

```diff
@@ -173,6 +179,10 @@ module.exports = {
         // Sanitize the job (note that jobs that do not implement the sanitize function
         // will not proceed
 
+        if (currentJob.payload.regression) {
+          process.env.REGRESSION = true;
```

Comment:
May I have the context on what this is intended to do?

Reply:
I think it would be better to pass this explicitly into the child environments, rather than using environment inheritance

Reply:
when we run our regressions, currently it overwrites the same bucket which beats the purpose of regressions. with this flag, we will be able to run the regressions so that the outputs are written to two different buckets and helps us in comparing the results. 

These are two webhooks for regressions 
https://webhooks.mongodb-stitch.com/api/client/v2.0/app/workerpoolstaging-qgeyp/service/githubRegressionTest/incoming_webhook/enqueueRegressionTestJobsInTest

https://webhooks.mongodb-stitch.com/api/client/v2.0/app/workerpool-boxgs/service/githubRegressionTest/incoming_webhook/enqueueRegressionTestJobsInProd

One operates out of prod and other operates out of staging. but they both treat it as STAGE job and writes it to the same bucket.

Reply:
We do pass the regression flag as part of the payload but the buckets are chosen in make file and i thought having it in environment is the easy way to get it accessed in make file

Reply:
We'd want to pass it in when we spawn the makefile process. But right now that would involve untangling some dragons, so this is fine I guess.